spiceaidocs/docs/components/data-accelerators/index.md

```diff
@@ -32,7 +32,7 @@ Supported Data Accelerators include:
 | --------------------------------- | ----------------------- | ------ | ---------------- |
 | [`duckdb`](./duckdb.md)           | Embedded DuckDB         | Beta   | `memory`, `file` |
 | [`postgres`](./postgres/index.md) | Attached PostgreSQL     | Beta   |                  |
-| [`arrow`](./arrow.md)             | In-Memory Arrow Records | Beta   | `memory`         |
+| [`arrow`](./arrow.md)             | In-Memory Arrow Records | RC     | `memory`         |
```

Comment:
Might be better `Release</br> Candidate` for readability?

Reply:
Let's copy from connectors and use `Release Candidate`, and make sure we change the table ordering so Release Candidate accelerators are at the top.

Reply:
@peasee  - updated 👍